### PR TITLE
Setup bitcoin mempool transactions pump

### PR DESCRIPTION
**Pull request by arturalbov**
Setup bitcoin mempool transactions pump

### CODE REVIEW

File: pumps/bitcoin/src/main/kotlin/fund/cyber/pump/bitcoin/client/BitcoinBlockchainInterface.kt [hunk 1]
```diff
@@ -29,15 +35,21 @@ class BitcoinBlockBundle(
...
-) : BlockchainInterface<BitcoinBlockBundle> {
+    private val bitcoinJsonRpcClient: BitcoinJsonRpcClient,
+    private val rpcToBundleEntitiesConverter: JsonRpcBlockToBitcoinBundleConverter,
+    monitoring: MeterRegistry,
+    private val mempoolHashesCache: Cache<String, String>
+) : BlockchainInterface<BitcoinBlockBundle>, PoolInterface<BitcoinTx> {
 
     private val downloadSpeedMonitor = monitoring.timer("pump_bundle_download")
 
+    private val mempoolCacheValue = ""
```
Comment (hleb-albau) [inline pumps/bitcoin/src/main/kotlin/fund/cyber/pump/bitcoin/client/BitcoinBlockchainInterface.kt]: wtf, leave comment

File: pumps/bitcoin/src/main/kotlin/fund/cyber/pump/bitcoin/client/BitcoinJsonRpcClient.kt [hunk 2]
```diff
@@ -49,23 +48,22 @@ class BitcoinJsonRpcClient(
...
-                    Flux.fromIterable(chunk)
-                        .flatMap { txId ->
-                            Mono.just(
-                                executeRestRequest("/rest/tx/$txId", JsonRpcBitcoinTransaction::class.java)
-                            ).subscribeOn(Schedulers.parallel())
-                        }
-                        .collectList().block()!!
-                }
+            return@recordCallable Flux.fromIterable(txIds)
+                .flatMap { hash -> getTxRx(hash).subscribeOn(Schedulers.parallel()) }
```
Comment (hleb-albau) [inline pumps/bitcoin/src/main/kotlin/fund/cyber/pump/bitcoin/client/BitcoinJsonRpcClient.kt]: newParallelol()

File: pumps/ethereum/src/main/kotlin/fund/cyber/pump/ethereum/client/EthereumBlockchainInterface.kt [hunk 3]
```diff
@@ -59,11 +62,15 @@ class EthereumBlockchainInterface(
...
-        parityClient.pendingTransactionObservable().toBlocking().subscribe(
-            { parityTx -> action(parityToBundleConverter.parityMempoolTxToDao(parityTx)) },
-            { error -> onError(error) }
-        )
+    override fun subscribePool(): Flowable<EthereumTx> {
+        return Flowable.create<EthereumTx>({ emitter ->
+
+            parityClient
+                .pendingTransactionObservable()
+                .flatMap { e -> Observable.just(parityToBundleConverter.parityMempoolTxToDao(e)) }
```
Comment (hleb-albau) [inline pumps/ethereum/src/main/kotlin/fund/cyber/pump/ethereum/client/EthereumBlockchainInterface.kt]: map?

**Comment by arturalbov** [comment]
Closes #171

**Pull request merged**